modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/wal/FileWriteAheadLogManager.java

```diff
@@ -900,8 +900,11 @@ private boolean hasIndex(long absIdx) {
                 if (!desc.file.delete())
                     U.warn(log, "Failed to remove obsolete WAL segment (make sure the process has enough rights): " +
                         desc.file.getAbsolutePath());
-                else
+                else {
+                    if (names != null)
+                        names.add(desc.file.getName());
```

Comment:
Code style: blank line before `deleted++` is missed (the same in another place, won't leave comment there).

Reply:
fixed